--- cairo-1.15.12/Makefile.in	2018-04-12 03:35:50.000000000 +0200
+++ cairo-1.15.12/Makefile.in.new	2018-06-21 14:58:39.443000000 +0200
@@ -107,7 +107,6 @@
 TESTS =
 check_PROGRAMS =
 # libpng is required for our test programs
-@CAIRO_HAS_PNG_FUNCTIONS_TRUE@am__append_1 = boilerplate test perf
 subdir = .
 ACLOCAL_M4 = $(top_srcdir)/aclocal.m4
 am__aclocal_m4_deps = $(top_srcdir)/build/aclocal.cairo.m4 \
@@ -718,7 +717,7 @@
 #	$(NULL)
 ACLOCAL_AMFLAGS = -I build ${ACLOCAL_FLAGS}
 DIST_SUBDIRS = src doc util boilerplate test perf
-SUBDIRS = src doc util $(am__append_1)
+SUBDIRS = src doc util
 DISTCHECK_CONFIGURE_FLAGS = \
 	--enable-gtk-doc \
 	--enable-test-surfaces \
